package/MDAnalysis/coordinates/DMS.py

```diff
@@ -70,7 +70,9 @@ def get_coordinates(self, cur):
         particles = cur.fetchall()
         return [(p['x'], p['y'], p['z']) for p in particles]
 
-    def get_particle_by_columns(self, cur, columns=['x', 'y', 'z']):
+    def get_particle_by_columns(self, cur, columns=None):
```

Comment:
@mnmelo Thanks for the feedback. We don't change your arguments with this rewrite as changing too much code would increase the risk of breaking your code. Replacing a list with a tuple might not always be safe.